web/client/components/export/ExportPanel.jsx

```diff
@@ -46,7 +47,7 @@ export default ({
                     <Glyphicon glyph="upload"/>
                 </div>
                 <div>
-                    <HTML msgId="mapExport.exportPanel.title"/>
+                    {exportPanelTitle}
```

Comment:
The implementation is good, but there is code that has been already changed in the dashboard PR see https://github.com/geosolutions-it/MapStore2/commit/ef79f2a534011b621b47fa125f9b3c1f630b381a#diff-99209a97ab18b88c088db5f5d51fb0deeb88efa3a95e813fa286477bab084f06R50

I expected to not see this highlighted.
Did you create the PR from a clean branch aligned with master?
Did you pull the current master on your branch?